DSpace/dspace-angular
# Fix for Creative Commons jurisdiction chosen is ignored

## References
* Fixes  #1836
* Requires DSpace/DSpace#[pr-number] 

## Description

This PR gets the default CC jurisdiction configured in the API key `cc.license.jurisdiction` and disables the jurisdiction selector if the value is different to none. When the key is set to none, it allows the user to select the jurisdiction and creates the CC URL based on the selection.

## Instructions for Reviewers

- This PR requires the REST API PR to expose the configuration key `cc.license.jurisdiction`.
- Activate the cclicense step in item-submission.xml 

Test1:
- Set the key `cc.license.jurisdiction` (it could be blank for International jurisdiction)
- Start a new submission and check that the jurisdiction selector is disabled and CC URL generated is based on the jurisdiction  configured

Test2:
- Set the value of `cc.license.jurisdiction` key to none
- Start a new submission and check that the jurisdiction selector is enabled and the CC URL generated is based on the jurisdiction selected. 

## Checklist
